Server/copy-assets.ts

```diff
@@ -0,0 +1,37 @@
+import * as path from 'path';
+import * as fs from 'fs';
+import * as util from 'util';
+
+const mkdir = util.promisify(fs.mkdir);
+const readdir = util.promisify(fs.readdir);
+const stat = util.promisify(fs.stat);
+const copyFile = util.promisify(fs.copyFile);
```

Comment:
If you use the fs/promises you don't have to create these costs but can use them right away https://nodejs.org/api/fs.html#fspromisescpsrc-dest-options

Reply:
adjusted